package.json

```diff
@@ -28,6 +28,7 @@
     "babel-core": "^5.2.17",
     "babel-eslint": "^4.0.5",
     "babelify": "^6.0.0",
+    "backbone.marionette": "^2.4.4",
```

Comment:
I'd rather not add Marionette as a strict dependency for this, since it can be used without Marionette, and the Marionette specific portions won't error without it, they just won't work.  It's possible I'm misunderstanding how to represent that in npm dependencies though.